### PR TITLE
Storybook v5

### LINKED ISSUE #7: Screenshot become no preview with storybook v5 in simple mode
## Overview
The screenshot was succeeded but became no preview when I used simple mode in react project.

This issue is related to #6.

## Details
Screenshots generated by zisui simple mode are all `no preview`

### Expected
Generated screenshots are correctly preview of each stories.

### Actual
`yarn zisui` commands succeeded but the generated screenshots are all `no preview`
like this.

with some emoji

## Reproduction

https://github.com/sasurau4/sample-zisui-storybook

### PR DESCRIPTION
**Pull request by Quramy**
It closes #7 , #8 . And also it may be related with #6 .

- [x] v4 simple mode
- [x] v4 managed mode
- [x] v5 simple mode
- [x] v5 managed mode

### CODE REVIEW

File: src/node/browser.ts [hunk 1]
```diff
@@ -97,18 +98,33 @@ export class StorybookBrowser extends Browser {
...
       await this.page.goto(this.opt.storybookUrl + "/iframe.html?selectedKind=zisui&selectedStory=zisui");
       await this.page.waitFor(() => (window as ExposedWindow).__STORYBOOK_CLIENT_API__);
-      stories = await this.page.evaluate(
-        () => (window as ExposedWindow).__STORYBOOK_CLIENT_API__.getStorybook().map(({ kind, stories }) => ({ kind, stories: stories.map(s => s.name) }))
+      [stories, oldStories] = await this.page.evaluate(
+        () => {
+          const win = window as ExposedWindow;
+          if (win.__STORYBOOK_CLIENT_API__.raw) {
+            // .raw API exsits only if storybook v5
+            return [win.__STORYBOOK_CLIENT_API__.raw().map(_ => ({ id: _.id, kind: _.kind, story: _.name, version: "v5"  })), null];
```
Comment (Quramy) [inline src/node/browser.ts]: `id` can be extracted with `raw()` api.

File: src/node/browser.ts [hunk 2]
```diff
@@ -275,23 +291,45 @@ $doc.body.appendChild($style);
...
+          }
+        };
+      case "v5" :
+        return {
+          key: "storybook-channel",
+          event: {
+            type: "setCurrentStory",
+            args: [
+              {
+                storyId: story.id,
```
Comment (Quramy) [inline src/node/browser.ts]: With storybook v5, we send `storyId` to storybook channel rather than `kind` and `story` .

**Comment by Quramy** [comment]
@heavenshell @sasurau4 how about this?

**Comment by heavenshell** [comment]
Thank you for fixing this!
As a quick review of this PR, I think this is good!
My patch is more workaround 😉

I'll checkout this PR and run at my local tomorrow 🙇

**Comment by Quramy** [comment]
I've added an example for v5 and managed-mode via 1fcd215 . 

But zisui throws `ScreenshotTimeoutError` ( It seems `emitCapture` does not work)

Reproduce it via:

```sh
cd examples/v5-managed-react
yarn
yarn zisui:all
```

File: src/types.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+export type StorybookVersin = "v4" | "v5";
```
Comment (heavenshell) [inline src/types.ts]: `s/StorybookVersin/StorybookVersion/`

Comment (Quramy) [reply src/types.ts]: Ooops 🙇

**Comment by Quramy** [comment]
> I've added an example for v5 and managed-mode via [1fcd215](https://github.com/Quramy/zisui/commit/1fcd2158523f6af387a7ecc6da7382acc48945d2) .
> 
> But zisui throws `ScreenshotTimeoutError` ( It seems `emitCapture` does not work)
> 

I fixed it via https://github.com/Quramy/zisui/pull/9/commits/b213a013c96e216ec86854bf0d25d5f4c94efcac#diff-227875b5788e4e39a19edacc3d70eeb0R29

File: src/node/browser.ts [hunk 4]
```diff
@@ -13,16 +13,22 @@ import {
...
   const q = parse(url).query || "";
-  const { selectedKind: kind, selectedStory: story } = querystring.parse(q);
-  if (!kind || Array.isArray(kind) || !story || Array.isArray(story)) return;
-  return { kind, story };
+  const { id, selectedKind: kind, selectedStory: story } = querystring.parse(q);
+  if (!id) {
+    if (!kind || Array.isArray(kind) || !story || Array.isArray(story)) return;
+    return `${kind}/${story}`;
+  } else {
+    if (Array.isArray(id)) return;
```
Comment (heavenshell) [inline src/node/browser.ts]: Yeah!! Perfect!!

**Comment by Quramy** [comment]
@heavenshell Thanks for your review 😄 

I will self-merge this PR if there are no serious pointed matter from you.

**Comment by heavenshell** [comment]
Thank you so much!!!!

**Comment by sasurau4** [comment]
Sorry for late response :bowing_man: 
It's awesome! Thanks for your speedy fixing!!! :tada: :tada: :tada:

**Pull request merged**